main.py

```diff
@@ -391,9 +391,10 @@ def output():
                     test = test.drop(columns="DiscardDestinationID")
                 except:
                     print("no column")
-                test.drop(test.tail(1).index, inplace=True)
-                print(test)
-                test = test.loc[:, ~test.columns.str.contains("^Unnamed")]
+                # print("pre test drop: ", test)
+                # test.drop(test.tail(1).index, inplace=True)
+                # test = test.loc[:, ~test.columns.str.contains("^Unnamed")]
+                print("post test drop: ", test)

```

Comment:
This is ok going into dev, but I want to start enforcing cleaning up print statements going into prd.